packages/web-components/fast-foundation/src/breadcrumb-item/breadcrumb-item.ts

```diff
@@ -0,0 +1,57 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+import { ARIAGlobalStatesAndProperties, StartEnd } from "../patterns/index";
+import { applyMixins } from "../utilities/apply-mixins";
+
+/**
+ * A Breadcrumb Item Custom HTML Element.
+ *
+ * @public
+ */
+export class BreadcrumbItem extends FASTElement {
+    /**
+     * The URL of the hyperlink
+     * @public
+     * @remarks
+     * HTML Attribute: href
+     */
+    @attr
+    public href: string;
+
+    /**
+     * @internal
+     */
+    @observable
+    public showSeparator: boolean = true;
+
+    /**
+     * @internal
+     */
+    @observable
+    public isCurrent: boolean = false;
+}
+
+/**
+ * Includes ARIA states and properties relating to the ARIA link role
+ *
+ * @public
+ */
+export class DelegatesARIABreadcrumbItem extends ARIAGlobalStatesAndProperties {
+    /**
+     * See {@link https://www.w3.org/WAI/PF/aria/roles#link} for more information
+     * @public
+     * @remarks
+     * HTML Attribute: aria-expanded
+     */
+    @attr({ attribute: "aria-expanded", mode: "fromView" })
+    public ariaExpanded: "true" | "false" | undefined;
+}
```

Comment:
Isn't this just a link? Should we just use the existing link definition? There's not an "ARIA breadcrumb item", I think this should just use the anchor class.

Reply:
yeah, I didn't think this was needed. will remove.